index.html

```diff
@@ -0,0 +1,265 @@
+                        ></a>
+                        <a href="#!" class="link-muted"
+                          ><i class="fas fa-redo-alt ms-2"></i
+                        ></a>
+                        <a href="#!" class="link-muted"
+                          ><i class="fas fa-heart ms-2"></i
+                        ></a>
+                      </div>
+                      <p class="mb-0">
+                        There are many variations of passages of Lorem Ipsum
+                        available, but the majority have suffered alteration in
+                        some form, by injected humour, or randomised words which
+                        don't look even slightly believable. If you are going to
+                        use a passage of Lorem Ipsum, you need to be sure.
+                      </p>
+                    </div>
+                  </div>
+                </div>
+
+                <hr class="my-0" />
+
+                <div class="card-body p-4">
+                  <div class="d-flex flex-start">
+                    <img
+                      class="rounded-circle shadow-1-strong me-3"
+                      src="https://mdbcdn.b-cdn.net/img/Photos/Avatars/img%20(24).webp"
+                      alt="avatar"
+                      width="60"
+                      height="60"
+                    />
+                    <div>
+                      <h6 class="fw-bold mb-1">Betty Walker</h6>
+                      <div class="d-flex align-items-center mb-3">
+                        <p class="mb-0">
+                          March 30, 2021
+                          <span class="badge bg-primary">Pending</span>
+                        </p>
+                        <a href="#!" class="link-muted"
+                          ><i class="fas fa-pencil-alt ms-2"></i
+                        ></a>
+                        <a href="#!" class="link-muted"
+                          ><i class="fas fa-redo-alt ms-2"></i
+                        ></a>
+                        <a href="#!" class="link-muted"
+                          ><i class="fas fa-heart ms-2"></i
+                        ></a>
+                      </div>
+                      <p class="mb-0">
+                        It uses a dictionary of over 200 Latin words, combined
+                        with a handful of model sentence structures, to generate
+                        Lorem Ipsum which looks reasonable. The generated Lorem
+                        Ipsum is therefore always free from repetition, injected
+                        humour, or non-characteristic words etc.
+                      </p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <div class="formGroup">
+        <label for="comment">Create a new post:</label>
+        <textarea class="form-control" rows="5" id="comment"></textarea>
+      </div>
```

Comment:
You should be using the `<form>` tag here.
You should also add a button for submitting the post.

Reply:
I have changed and fixed them now.

Reply:
Your changes look good. You can resolve this conversation.